### DISCUSSION TITLE
Need help in editor

### DISCUSSION THREAD

**Discussion by BestiaPL** [FarGroup/FarManager#572] [Q&A] [answered]
Hello.

I'm using Far with FRename3 plugin [https://forum.farmanager.com/viewtopic.php?f=39&t=9719](url).
After finding some folders/files in standard Search dialog I can select and with FRename3 edit names - i.e. change Polish national characters into English.
I can mark vertical block of text (using shortcut from FRename3), but when I try to Search/Replace text Far is replacing in whole editor, not only in selected block.
Maybe I'm wrong - so, if there is an option to enable - please tell me where is it :)

Screen from Far/FRename3 with selected block (in Polish language).
![far_editor](https://user-images.githubusercontent.com/26287042/207862750-3a4a0ff3-3c07-4f90-b431-dd85cc37d953.JPG)

**Comment by yegor-mialyk** [comment, marked as answer]
Far Manager does not support search/replace in selected text, but you can use a 3rd party plugin to achieve this, e.g. LuaFAR Search.

**Reply by BestiaPL** [reply]
Big thank!! This is it :)
